FilteringApp/FilteringApp.ConsoleUI/Program.cs

```diff
@@ -1,58 +1,114 @@
 ﻿using FilteringApp.Core.Models;
 using FilteringApp.Core.Utils;
+using FilteringApp.Core.Other;
 using System;
 using System.Collections.Generic;
 
+
 namespace FilteringApp.ConsoleUI
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.Clear();
-            int[] array = new int[0];
-            var isArrayValid = false;
-            var arrayPackage = new ArrayParseResult(array);
-            while (isArrayValid == false)
+            FilterTheArray();
+            SumTheArray();
+            ArrayGetMinMax();
+            InstanceCountItself();              
+        }
+
+        static void FilterTheArray()
```

Comment:
Always explicitly type visibility of the method, "private" or "public" or smth else

Reply:
done